### PR TITLE
Clean codebase

### PR DESCRIPTION
**Pull request by madcake**

### CODE REVIEW

**Review by hewigovens — APPROVED** [review]
lgtm, integration test was failed

File: api/endpoint/tickers.go [hunk 1]
```diff
@@ -144,14 +174,25 @@ func handleTickersError(c *gin.Context, req controllers.TickerRequest) {
...
 		if _, value := keys[entry]; !value {
 			keys[entry] = true
-			list = append(list, entry)
+			result = append(result, entry)
+		}
+	}
+	return result
+}
+
+func parseAssetIds(ids []string) (assets []controllers.Asset) {
```
[the remark this answers is not in the document]
Comment (madcake) [reply api/endpoint/tickers.go:188]: Yeah

File: api/endpoint/tickers.go [hunk 2]
```diff
@@ -76,20 +82,26 @@ func GetTickerHandlerV2(controller controllers.TickersController) func(c *gin.Co
 // @Router /v2/market/tickers [post]
 func PostTickersHandlerV2(controller controllers.TickersController) func(c *gin.Context) {
 	return func(c *gin.Context) {
-		request := controllers.TickerRequestV2{Currency: watchmarket.DefaultCurrency}
+		var request struct {
```
Comment (vikmeup) [inline api/endpoint/tickers.go:85]: can you move request to models?

File: api/endpoint/tickers.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (vikmeup) [reply api/endpoint/tickers.go:188]: +1 to have this on golibs

File: services/controllers/tickers/base.go [hunk 3]
```diff
@@ -34,34 +36,159 @@ func NewController(
...
+		c.applyRateToTicker(&ticker, r)
+		result = append(result, ticker)
+	}
+	return result
 }
 
-func (c Controller) HandleTickersRequest(tr controllers.TickerRequest) (controllers.TickerResponse, error) {
-	rate, err := c.getRateByPriority(strings.ToUpper(tr.Currency))
+func (c Controller) getTickersByPriority(assets []controllers.Asset) (watchmarket.Tickers, error) {
+	if c.configuration.RestAPI.UseMemoryCache {
```
Comment (vikmeup) [inline services/controllers/tickers/base.go:69]: We always have `UseMemoryCache` as true, I think we can remove anything in `else`

Comment (vikmeup) [reply services/controllers/tickers/base.go:69]: we can even remove `UseMemoryCache` property as it's always in memory

**Review by vikmeup — APPROVED** [review]

**Review by vikmeup — APPROVED** [review]

**Review by hewigovens — APPROVED** [review]

**Pull request merged**